src/relay/crypto.js

```diff
@@ -11,7 +11,14 @@ export const constructStealthPubKey = function (emphemeralPrivKey, destPubKey) {
   let digestPublicKey = PrivateKey.fromBuffer(digest).toPublicKey() // H(ebG)G
 
   let stealthPublicKey = PublicKey(digestPublicKey.point.add(destPubKey.point)) // H(ebG)G + bG
-  return stealthPublicKey
+  return new cashlib.HDPublicKey({
+    publicKey: stealthPublicKey.toBuffer(),
+    depth: 0,
+    network: 'testnet',
+    childIndex: 0,
+    chainCode: digest.slice(0, 32),
```

Comment:
I'm not sure using the `digest.slice(0,32)` (~`digestPrivateKey`) as a chainCode here is a good idea. Perhaps we should hash the digest again.

Reply:
It should not be a problem